### DISCUSSION TITLE
jenkins

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1604] [Q&A] [answered]
![Screenshot 2023-11-14 221457](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/5912b185-f99f-4ecb-a2ce-d6a3764dcd37)
I failed installation of jenkins, is it any problem?
I failure because of without doing previous classes practice shell, ansible, I attended all classes, some gap is there on practice, and also  i faced technical issues.

**Comment by sreedharm07** [comment, marked as answer]
To check the error you dint mentioned any ip, please mention ip when you raise issue, to check th error.

anyway, from the screen shot the error was shell command in step **execute shell**  was wrong thats why its showing command not found.
please give command  correctly.
